recipes/flynt/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "flynt" %}
+{% set version = "0.78" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flynt-{{ version }}.tar.gz
+  sha256: 15082167a538ea56b6ce4ca6874c22c739710759832dda78d8f43b6079b7e646
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - flynt = flynt:main
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+    - astor
+    # want to stay noarch: python so adding this for all python
+    - tomli >=1.1.0
+
+test:
+  imports:
+    - flynt
+  commands:
+    - pip check
+    - flynt --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ikamensh/flynt
+  summary: CLI tool to convert a python project's %-formatted strings to f-strings.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  dev_url: https://github.com/ikamensh/flynt
+  doc_url: https://github.com/ikamensh/flynt
+
+extra:
+  recipe-maintainers:
+    - altheaden
+    - xylar
```

Comment:
@altheaden, please respond that you're okay to be a maintainer

Reply:
Yep!